strings-csharp/RandomAlphaNumericString/RandomAlphaNumericString/CryptographicallySecureMethods/AesMethod.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Security.Cryptography;
+
+namespace RandomAlphaNumericString
+{
+    public partial class Methods
+    {
+        //Maximal Length: 44
+        public static string? AesMethod(int length)
+        {
+            using var crypto = Aes.Create();
+            crypto.GenerateKey();
+            return Convert.ToBase64String(crypto.Key).Substring(0, length);
+        }
```

Comment:
This method is not producing only alpha-numeric strings, unless we are clear that both a '+', '/' and '=' are considered alpha-numeric.